packages/api-client/src/main/shims/node/cookie.ts

```diff
@@ -24,18 +24,16 @@ import {Cookie as ToughCookie} from 'tough-cookie';
 import {AccessTokenData, AUTH_COOKIE_KEY, AUTH_TABLE_NAME, Cookie} from '../../auth';
 import {HttpClient} from '../../http';
 
-const COOKIE_NAME: string = 'zuid';
-
-type PersistedCookie = {
+interface PersistedCookie {
   expiration: string;
   zuid: string;
-};
+}
 
-const loadExistingCookie = async (engine: CRUDEngine): Promise<Cookie> => {
+const loadExistingCookie = (engine: CRUDEngine): Promise<Cookie> => {
   return engine
     .read<PersistedCookie>(AUTH_TABLE_NAME, AUTH_COOKIE_KEY)
-    .catch(error => {
-      if (error instanceof StoreEngineError.RecordNotFoundError) {
+    .catch((error: Error) => {
+      if (error instanceof StoreEngineError.RecordNotFoundError || error.constructor.name === 'RecordNotFoundError') {
```

Comment:
When `error instanceof StoreEngineError.RecordNotFoundError` is not working at all then we should always take `error.constructor.name === 'RecordNotFoundError'`.

Reply:
Actually we do the same in Account.ts:
https://github.com/wireapp/wire-web-packages/blob/3d2effa274e186d0c54b0c969f20c2b64b1736da/packages/core/src/main/Account.ts#L141-L146

So I vote for checking both with `instanceof` and `error.constructor.name`.

Reply:
But there one check should be sufficient, no?